### ISSUE TITLE
Feature request: fast adding of tracked time via selecting a time period on the day log

### ISSUE THREAD

**Issue by 0nxku** [haydenull/logseq-plugin-agenda#258]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

I think it would be very useful to add tracked time via the time selection instead of it (as a second mode).

But for a precise addition of tracked time through the time selection on the day log gaps for selection have to be smaller. Now it's 15 minutes. It would be great to have the ability to change the scale so that you can select your time to the minutes.

### Reproduce the Bug

Currently, when you select a period of time on the calendar day log, you select a start date and estimated time.

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by haydenull** [comment]
The input box here allows manual input of any time.

<img width="626" alt="image" src="https://github.com/haydenull/logseq-plugin-agenda/assets/17695989/09e9962e-bb45-4ffe-b67a-a8ed17484a37">

**Comment by 0nxku** [comment]
Yes, I understand, but it's not super convenient when you're trying to track all of your time, you have to do many this many many times.
